src/components/dls/PopoverMenu/PopoverMenu.tsx

```diff
@@ -44,6 +45,7 @@ PopoverMenu.Item = ({
   onClick,
   isDisabled,
   shouldCloseMenuAfterClick = false,
+  shouldFlipIconRTL = false,
```

Comment:
nit: I think it should be renamed to `shouldFlipOnRTL` which is already used in multiple places